microcosm_postgres/migrate.py

```diff
@@ -0,0 +1,203 @@
+"""
```

Comment:
The bulk of the changes are in this module; it wraps Alembic in such a way that most configuration can be avoided.

This comes at a cost of some potentially fragile patching of the Alembic `ScriptDirectory` class. The payoff is that we can use conventions for most of the configuration choices and just ship migration files within a migrations directory of each source tree (so that migrations ship with source). I think this is an ok trade-off.